Creating markdown file for decrement Exact (#1145)
Codecademy/docs: content/java/concepts/math-methods/terms/decrement-exact/decrement-exact.md

---

> Title: '.decrementExact()'
> 
> Description: 'Returns the argument decremented by one, throwing an exception if the result overflows the datatype.'
> Subjects:
> - 'Computer Science'
>   Tags:
> - 'Functions'
> - 'Methods'
> - 'Arithmetic'
> 
>   CatalogContent:
> - 'learn-java'

Reviewer: ```suggestion
Title: '.decrementExact()'
Description: 'Returns the argument decremented by one, throwing an exception if the result overflows the datatype.'
Subjects:
- 'Computer Science'
  Tags:
- 'Functions'
- 'Methods'
- 'Arithmetic'
  CatalogContent:
- 'learn-java'
```

Empty lines aren't necessary in the entry meta-data.

---

> The **`Math.decrementExact()`** method returns the argument decremented by one. It throws an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows the specified datatype either `long` or `int`.

Reviewer: ```suggestion
The **`Math.decrementExact()`** method returns the argument decremented by one. It throws an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows the specified [data type](https://www.codecademy.com/resources/docs/java/data-types) (`long` or `int`) of the argument.
```

## Syntax

```pseudo
Math.decrementExact(num)
```

> - The `num` argument is a `long` or `int` value.

Reviewer: ```suggestion
The `num` parameter is of type `long` or `int`.
```

## Example

The following example uses `Math.decrementExact()` to decrement the input by 1:

```java

> public class Main {

Reviewer: ```suggestion
// Main.java

public class Main {
```
